lms/djangoapps/courseware/user_state_client.py

```diff
@@ -41,6 +39,9 @@ class DjangoXBlockUserStateClient(XBlockUserStateClient):
         even though the actual stored state in the database will be ``"{}"``).
     """
 
+    # Use this sample rate for DataDog events.
+    API_DATADOG_SAMPLE_RATE = 0.01
```

Comment:
It's called `ConfigurationModel` - but it's too heavyweight to use here IMO - it'd require a migration. I'm going to punt and use this hardcoded sample rate.